run.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cd docs && make dockerhtml &
+
+wait
```

Comment:
Don't suppose we can fold this inside the Dockerfile and drop the file?

Reply:
I still think this is the best way to handle this since it's by far the easiest. It also avoids creating permissions and allows for easier extensibility. Since you clearly disagree, I'll just go ahead and CLOSE this ticket.